plugins/woocommerce-admin/client/marketplace/components/product-list/types.ts

```diff
@@ -29,3 +30,9 @@ export interface Product {
 	averageRating?: number | null;
 	reviewsCount?: number | null;
 }
+
+export enum ProductType {
```

Comment:
I'm uncertain what the difference is between `extensions` and `classic`. Wouldn't they be the same?

Reply:
They are the same now, `classical` should eventually be removed.